omicsclaw.py

```
@@ -1204,6 +1222,135 @@ def main():
+                script_path = skill_info.get("script")
+                if script_path and Path(script_path).exists():
+                    try:
+                        import importlib.util as _ilu
+                        _spec = _ilu.spec_from_file_location("_skill_mod", str(script_path))
+                        _mod = _ilu.module_from_spec(_spec)
+                        _spec.loader.exec_module(_mod)  # type: ignore[union-attr]
+                        r_enhanced_plots = getattr(_mod, "R_ENHANCED_PLOTS", {})
+                        if renderer in r_enhanced_plots:
+                            filename = r_enhanced_plots[renderer]
+                    except Exception:
+                        pass  # fall back to default filename
```

Ruff (0.15.9)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)